fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1807,6 +1808,11 @@ public Void visitDictQueryExpr(DictQueryExpr node, Scope context) {
             node.setDictQueryExpr(dictQueryExpr);
             return null;
         }
+
+        @Override
+        public Void visitParameterExpr(Parameter node, Scope context) {
+            return null;
```

Comment:
should call actual expr's analyze here

Reply:
but other expression will depend on child's class type(like lambda), I think it's can't work good, and will cause some expcetion